pkg/karmadactl/cmdinit/kubernetes/rbac.go

```diff
@@ -1,60 +1,15 @@
 package kubernetes
 
 import (
-	"context"
-	"fmt"
+	"github.com/karmada-io/karmada/pkg/karmadactl/cmdinit/utils"
 
 	rbacv1 "k8s.io/api/rbac/v1"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"k8s.io/klog/v2"
 )
 
-// ClusterRoleFromSpec ClusterRole spec
-func (i *CommandInitOption) ClusterRoleFromSpec(name string, rules []rbacv1.PolicyRule) *rbacv1.ClusterRole {
```

Comment:
I'm glad to see moving this kind of method out of the `option` structure, they shouldn't be here.